src/pages/eats/dashboard.hbs

```diff
@@ -0,0 +1,143 @@
+          {{#each settings.dashboard.categories}}
+          <div class="col-md-6 col-lg-4">
+            <div class="bxi-categories mb-4 d-flex p-4 flex-column align-items-start overflow-hidden position-relative">
+              <div class="img-wrapper"><img class="position-absolute top-50 start-50 translate-middle bxi-img-cover" src="{{background_image}}" /></div>
+              <h1 class="position-relative bxi-categories-section--text bxi-zi-2 fw-bold mb-0">{{percentage}}</h1>
+              <h5 class="position-relative bxi-categories-section--text bxi-zi-2 fw-bold mb-5">{{category}}</h5>
+              <button class="bxi-button bxi-bg-primary fw-bold position-relative mt-5 mb-4 py-3 px-5">{{button}}</button>
+            </div>
+          </div>
+          {{/each}}
+        </div>
+        <div class="row mt-5">
+          <div class="col-lg-7 col-xl-8 mb-5">
+            <div class="bxi-eats-portal rounded-2 mb-4 p-2">
+              <h1 class="bxi-eats-portal--header bxi-ff-secondary mt-3 ms-4 mb-4">{{settings.dashboard.portal.title}}</h1>
+              {{#each settings.dashboard.portal.orders}}
+              <div class="d-flex justify-content-between bxi-checklist-item py-3 px-5">
+                <div class="d-flex align-items-center">
+                  <div class="bxi-checkmark d-flex justify-content-center align-items-center rounded-circle bxi-checkmark--checked">
+                    {{> checkmarkIcon}}
+                  </div>
+                  <div class="ms-4">
+                    <h5 class="bxi-eats-dashboard--color fw-bold mb-0">{{date}}</h5>
+                    <p class="bxi-eats-portal--text mb-0">{{items}}</p>
+                  </div>
+                </div>
+                <div class="bxi-document d-flex flex-nowrap align-items-center">
+                  {{> documentIcon class='me-0'}}
+                  <span class="d-inline-block ms-2 bxi-document--color ms-0">{{button}}</span>
+                </div>
+              </div>
+              {{/each}}
+            </div>
+            {{#if env.BXI_DASHBOARD_POLICY_ID}}
+            <div class="mb-3" data-dv-flow="static" data-policy-id="{{env.BXI_DASHBOARD_POLICY_ID}}"></div>
+            {{/if}}
+          </div>
+          <div class="col-lg-5 col-xl-4">
+            <div class="row">
+              <div class="col-sm-12 col-md-4 col-lg-12 bxi-eats-sidebar rounded-2 mb-4">
+                <h1 class="bxi-eats-portal--header bxi-eats-dashboard--color bxi-ff-secondary fw-bold ms-0 mt-4 text-center ">{{settings.dashboard.sidebar.title}}</h1>
+                <div class="bxi-finance-card text-center bxi-bg-primary pb-1 mt-4">
+                  <p class="bxi-finance-card--description pt-3 mb-1">{{settings.dashboard.sidebar.card.description}}</p>
+                  <h1 class="bxi-main-header bxi-ff-secondary">{{settings.dashboard.sidebar.card.amount}}</h1>
+                  <div class="bxi-finance-card--line mb-4"></div>
+                </div>
```

Comment:
Let's change this to a real link just in case users ever want to use it.

`<a href="#" class="bxi-link text-center fw-bold d-block my-5 mx-auto">{{settings.dashboard.sidebar.card.link}}</a>`